src/hatch/cli/application.py

```diff
@@ -160,9 +160,9 @@ def ensure_plugin_dependencies(self, dependencies: list[Requirement], *, wait_me
             if dependencies_in_sync(dependencies):
                 return
 
-            pip_command = [sys.executable, '-u', '-m', 'pip']
+            pip_command = [sys.executable, '-u', '-m', 'uv', 'pip']
```

Comment:
`uv` makes a singular Python import available, `find_uv_bin` - that may be more reliable than `python -m uv ...` 🤔 
https://github.com/astral-sh/uv/blob/54b3a438d03483c6e76f05168bd417657b693e4b/python/uv/_find_uv.py#L8C5-L8C16

This likely works as well though

Reply:
Updated!